### PR TITLE
LG-4505: Update Atlas Logo

### PR DESCRIPTION
**Pull request by bruugey**
## ✍️ Proposed changes

🎟 _Jira ticket:_ [LG-4505](https://jira.mongodb.org/browse/LG-4505)

### For bug fixes, new features & breaking changes

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have run `yarn changeset` and documented my changes

### For new components

- [ ] I have added my new package to the global tsconfig
- [ ] I have added my new package to the Table of Contents on the global README
- [ ] I have verified the Live Example looks as intended on the design website.

### CODE REVIEW

**Review by TheSonOfThomp — APPROVED** [review]

**Review by TheSonOfThomp — COMMENTED** [review]
Missed that there's no changeset, bot otherwise LGTM

File: packages/logo/src/Logos/AtlasLogoMark.tsx [hunk 1]
```diff
@@ -1,170 +1,97 @@
...
+import { palette } from '@leafygreen-ui/palette';
 
-import { ProductLogoProps } from '../Logo.types';
-import { getAccessibleProps, getColor } from '../utils';
+import { ProductLogoProps, SupportedColorsMap } from '../Logo.types';
+import { getAccessibleProps } from '../utils';
 
 /**
  * @deprecated
  */
```
Comment (TheSonOfThomp) [inline packages/logo/src/Logos/AtlasLogoMark.tsx:11]: this should be "undeprecated" too

File: packages/logo/src/Logos/AtlasLogoMark.tsx [hunk 2]
```diff
@@ -1,170 +1,97 @@
...
     return (
       <svg
         {...getAccessibleProps({ role, 'aria-label': ariaLabel })}
-        x="0px"
-        y="0px"
-        viewBox="0 0 18 18"
-        width={size}
-        height={size}
+        width="179"
+        height="33"
```
Comment (TheSonOfThomp) [inline packages/logo/src/Logos/AtlasLogoMark.tsx:35]: should this also be `height` ?

**Review by sandynguyenn — APPROVED** [review]

File: packages/logo/src/Logos/AtlasLogoMark.tsx (hunk 2 above)
Comment (bruugey) [reply packages/logo/src/Logos/AtlasLogoMark.tsx]: Updated!

File: packages/logo/src/Logos/AtlasLogoMark.tsx (hunk 1 above)
Comment (bruugey) [reply packages/logo/src/Logos/AtlasLogoMark.tsx]: Updated!

File: packages/logo/src/Logo.stories.tsx [hunk 3]
```diff
@@ -153,12 +153,11 @@ export const CommunityEditionLockup = (args: BaseLogoProps) =>
   Template(CommunityEditionLogoLockup, args);
 export const UniversityLockup = (args: BaseLogoProps) =>
   Template(UniversityLogoLockup, args);
+export const AtlasMark = (args: BaseLogoProps) => Template(AtlasLogoMark, args);
+AtlasMark.storyName = 'Atlas Mark';
```
Comment (TheSonOfThomp) [inline packages/logo/src/Logo.stories.tsx:157]: Thinking about this more, this shouldn't be the `AtlasMark`. This is an updated `Atlas` logo

Comment (TheSonOfThomp) [reply packages/logo/src/Logo.stories.tsx:157]: Also, I believe all the `...Lockup` logos are deprecated (@sandynguyenn can you confirm? )

File: packages/logo/src/Logos/AtlasLogoMark.tsx [hunk 4]
```diff
@@ -1,170 +1,93 @@
...
+
     return (
       <svg
         {...getAccessibleProps({ role, 'aria-label': ariaLabel })}
-        x="0px"
-        y="0px"
-        viewBox="0 0 18 18"
-        width={size}
-        height={size}
+        height={height}
```
Comment (TheSonOfThomp) [inline packages/logo/src/Logos/AtlasLogoMark.tsx:31]: I was wrong! This should match the `viewBox` hard-coded numbers (otherwise the logo will warp!)

**Pull request closed without merging**